### ISSUE TITLE
Doesnt work with latest game patch

### ISSUE THREAD

**Issue by xiaoxiao921** [Theray070696/RoR2TierSelection#7]
SetItemDefs hook -> RoR2.ItemDef.tier is not a field anymore

it kills game loading

bump your game assembly references to latest

also your readme on thunderstore leads and tell users to use a list meant for devs only (debug only)
`[Here's a handy list of the internal names and their respective names](https://github.com/risk-of-thunder/R2Wiki/wiki/Item-&-Equipment-IDs-and-Names)`

the url got changed on the wiki, its only there for dev reference, if you need user to use internal name, tell them to use the console command or just change the system and don't rely on such thing

**Comment by Theray070696** [comment]
I'll update the mod at some point this weekend

**Comment by Theray070696** [comment]
Well shit, you meant just recompile lmao. I kept putting off updating this because I thought it would be more work than a recompile. Bug is fixed, will upload to Thunderstore soon-ish.

As for the link, I used to have it using the normal names, but around the first bugfix for 1.0, I had to move the tier changing to earlier in the loading process, which put it before translations were loaded, so internal names are what I'm unfortunately stuck with.
